### PR TITLE
Implement deleverage

### PR DESCRIPTION
**Pull request by hihiben**
- [x] deleverage logic
- [x] unit test
- [x] optional parameter fix in leverage and collateral swap
- [x] update tests without optional parameters

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export * from './api';
 export * from './config';
 export * from './leverage';
+export * from './deleverage';
```
Comment (zodahu) [inline src/index.ts:4]: @Jeff-CCH Should collateral-swap be added `infex.ts`? If yes please Ben help fix in this PR.

Comment (Jeff-CCH) [reply src/index.ts:4]: Yes. Please help fix it. Thanks.

File: src/deleverage.test.ts [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+      'utilization',
+      'healthRate',
+      'liquidationThreshold',
+      'borrowUSD',
+      'collateralUSD',
+      'netAPR'
+    );
+  });
+
+  it('Test buildLeverageTransactionRequest', async function () {
```
Comment (Jeff-CCH) [inline src/deleverage.test.ts]:
```suggestion
  it('Test buildDeleverageTransactionRequest', async function () {
```

**Comment by zodahu** [comment]
LGTM

**Review by Jeff-CCH — APPROVED** [review]

**Comment by Jeff-CCH** [comment]
LGTM

**Pull request merged**